Resources/vrecord_functions

```diff
@@ -1,6 +1,17 @@
 # these functions are appended to an include file that is sent to gtkdialog and allows for functions to be used within gtkdialog actions
 
 # functions
+_query_os_type(){
+    QUERY_OS_TYPE="$(uname -s)"
+    if [[ "${QUERY_OS_TYPE}" = "Darwin" ]] ; then
+        OS_TYPE="macOS"
+    elif [[ "${QUERY_OS_TYPE}" = "Linux" ]] ; then
+        OS_TYPE="linux"
+    fi
+}
+
+_query_os_type
```

Comment:
Is this a copy & paste survivor?

Reply:
hi @retokromer no, that was intentional for some testing, but will get rebased out soon. Doing some experiments to figure out some differences in shell behavior when it is called by GTK Dialog on Mac vs Linux. Will clean it all up prior to merge